.github/workflows/tests.yml

```
@@ -69,7 +69,7 @@ jobs:
           restore-keys: ${{ steps.extcache.outputs.key }}
 
       - name: "Install PHP"
-        uses: "shivammathur/setup-php@v2"
+        uses: "alcaeus/setup-php@allow-github-extension-installs"
```

This will be updated once the changes has been released as part of v2.

---

Ah, I hadn't thought of develop. In that case, I'd suggest this:
```suggestion
        # TODO: switch back to @v2 once a tag has been created
        uses: "shivammathur/setup-php@develop"
```

---

Changed it to use `@develop`.